server/src/test/resources/application.properties

```diff
@@ -10,11 +10,12 @@ kitu.kotoutumiskoulutus.koealusta.wstoken=
 kitu.kotoutumiskoulutus.koealusta.baseurl=http://localhost:8080/dev/koto/
 kitu.kotoutumiskoulutus.koealusta.scheduling.enabled=false
 
-kitu.oppijanumero.username=koto-rekisteri
-kitu.oppijanumero.password=
-kitu.oppijanumero.callerid=
-kitu.oppijanumero.casUrl=
-kitu.oppijanumero.serviceUrl=
+kitu.oppijanumero.username=username
+kitu.oppijanumero.password=password
+kitu.oppijanumero.callerid=callerid
+kitu.oppijanumero.casUrl=http://localhost:8080/dev/cas
+kitu.oppijanumero.service.url=https://localhost:8080/dev/oppijanumerorekisteri-service
```

Comment:
Elks testissä pitäis just käyttää mockia?

Reply:
Se ei käytä näitä tietoja.

Reply:
poistettu